lib/cadet/assessments.ex

```diff
@@ -0,0 +1,590 @@
+defmodule Cadet.Assessments do
+  @moduledoc false
+  import Ecto.Changeset
+  import Ecto.Query
+  import Cadet.ContextHelper
+
+  alias Timex.Timezone
+  alias Timex.Duration
+
+  alias Cadet.Repo
+
+  alias Cadet.Assessments.Mission
+  alias Cadet.Assessments.Question
+  alias Cadet.Assessments.Answer
+  alias Cadet.Assessments.Submission
+  alias Cadet.Accounts.User
+  alias Cadet.Course.Group
+
+  def all_missions, do: Repo.all(Mission)
+
+  def all_missions(category) do
+    Repo.all(from(a in Mission, where: a.category == ^category))
+  end
+
+  def all_open_missions(category) do
+    now = Timex.now()
+
+    mission_with_category = Repo.all(from(a in Mission, where: a.category == ^category))
+    Enum.filter(mission_with_category, &(&1.is_published && Timex.before?(&1.open_at, now)))
+  end
+
+  def missions_due_soon() do
+    now = Timex.now()
+    week_after = Timex.add(now, Duration.from_weeks(1))
+
+    all_missions()
+    |> Enum.filter(
+      &(&1.is_published && Timex.before?(&1.open_at, now) &&
```

Comment:
use `and` instead of `&&` similarly for the rest of the file. Refer to https://github.com/lexmag/elixir-style-guide#boolean-operators